packages/eslint-plugin/src/rules/switch-exhaustiveness-check.ts

```diff
@@ -14,9 +14,11 @@ import {
   requiresQuoting,
 } from '../util';
 
+const DEFAULT_COMMENT_PATTERN = /^no default$/iu;
```

Comment:
[Feature] Actually, now that I'm re-reading this, I'm not sure we should have a default here. The core ESLint rule does it but it seems kind of arbitrary to me. Maybe we should just have no default at all?

cc @typescript-eslint/triage-team

Reply:
I think 
- "no default" is a sensible default ignore comment pattern. 
- Sensible defaults are good things since they help standardize across projects (and in this case even between rules)
- I probably wouldn't have any comment ignore pattern on-by-default though
- Making a separate on/off toggle for the comment ignore pattern and a regexp option for it is overly verbose
- Alignment with [default-case](https://eslint.org/docs/latest/rules/default-case) is good for user experience

Therefore probably having the single regexp option that defaults to "no default" is the best compromise option here. But no strong opinion.

Reply:
👍 I'm ok with this. Your reasoning is convincing to me, in particular the alignment with the core rule.